morango/utils/syncing_utils.py

```diff
@@ -0,0 +1,14 @@
+_syncing_models = {}  # dictionary of models with morango naming identifier
+
+
+def add_syncing_models():
+    from morango.models import SyncableModel
+    import django.apps
+
+    model_list = []
+    for model_class in django.apps.apps.get_models():
+        if issubclass(model_class, SyncableModel) and not model_class._meta.proxy:
+            model_list.append(model_class)
+
+    for model_class in model_list:
+        _syncing_models.update({model_class.morango_model_name: model_class})
```

Comment:
I've fleshed this out more because now I have a dictionary based on profile. I have also ordered models in those profiles, based on foreign key dependencies.